src/common/meta/src/ddl/utils.rs

```diff
@@ -27,19 +27,17 @@ use crate::key::TableMetadataManagerRef;
 use crate::peer::Peer;
 use crate::rpc::ddl::CreateTableTask;
 
-pub fn handle_operate_region_error(datanode: Peer) -> impl FnOnce(Error) -> Error {
+/// Adds [Peer] context if the error is unretryable.
+pub fn add_peer_context_if_need(datanode: Peer) -> impl FnOnce(Error) -> Error {
```

Comment:
Perhaps "add_peer_context_if_needed" or "add_peer_context_as_needed" would be more suitable names.